.devcontainer/devcontainer.json

```
@@ -0,0 +1,32 @@
+{
+    "name": "Dev Container",
+    "build": {
+        "dockerfile": "../src/Dockerfile",
+        "context": "../src/",
+        "args": {
+            "IS_DEV_CONTAINER": "true",
+            "SKIPGC": "false"
+        }
+    },
+    "forwardPorts": [
+        8080
+    ],
+    "customizations": {
+        "vscode": {
+            "settings": {
+                "python.defaultInterpreterPath": "/usr/local/bin/python",
+                "python.linting.enabled": true,
+                "python.linting.pylintEnabled": true
+            },
+            "extensions": [
+                "ms-python.python",
+                "ms-python.vscode-pylance"
+            ]
+        },
+        "codespaces": {
+            "openFiles": [
+                "README.md"
+            ]
+        }
```

Do we need this one? Do we want a default open while?

If we need one, it might be better to open this one:

```suggestion
        "codespaces": {
            "openFiles": [
                "src/README.md"
            ]
        }
```

---

Yes, makes sense. I also moved a badge there.
I like to be reminded of guidelines when developing something (existing ones and potential for improvement).